pkg/store/change/change_test.go

```diff
@@ -141,50 +139,41 @@ func Test_changecreation(t *testing.T) {
 
 func Test_badpath(t *testing.T) {
 	badpath := "does_not_have_any_slash"
-	conf1, err1 := NewChangeValue(badpath, types.NewTypedValueString("123"), false)
+	conf1, err1 := types.NewChangeValue(badpath, types.NewTypedValueString("123"), false)
 
 	assert.Error(t, err1, badpath, "Expected error on ", badpath)
 
 	assert.Assert(t, conf1 == nil, "Expected config to be empty on error")
 
 	badpath = "//two/contiguous/slashes"
-	_, err2 := NewChangeValue(badpath, types.NewTypedValueString("123"), false)
+	_, err2 := types.NewChangeValue(badpath, types.NewTypedValueString("123"), false)
 	assert.ErrorContains(t, err2, badpath, "Expected error on path", badpath)
 
 	badpath = "/test*"
-	_, err3 := NewChangeValue(badpath, types.NewTypedValueString("123"), false)
+	_, err3 := types.NewChangeValue(badpath, types.NewTypedValueString("123"), false)
 	assert.ErrorContains(t, err3, badpath, "Expected error on path", badpath)
 }
 
 func Test_changeValueString(t *testing.T) {
-	cv1, _ := NewChangeValue(Test1Cont1ACont2ALeaf2A, types.NewTypedValueUint64(123), false)
-	var buf [8]byte
-	binary.LittleEndian.PutUint64(buf[:], 123)
-	byteArr123 := fmt.Sprintf("%v", buf)
+	cv1, _ := types.NewChangeValue(Test1Cont1ACont2ALeaf2A, types.NewTypedValueUint64(123), false)
 
-	assert.Equal(t, cv1.String(), Test1Cont1ACont2ALeaf2A+" "+byteArr123+" false",
+	assert.Equal(t, cv1.String(), `path:"/cont1a/cont2a/leaf2a" value:<bytes:"{\000\000\000\000\000\000\000" type:UINT > `,
```

Comment:
same

Reply:
this is the test of the String() function